.storybook/pages/ProjectOverview/ProjectOverview.tsx

```diff
@@ -69,13 +69,13 @@ export const ProjectOverview = ({ activeIndex = 0 }: Props) => {
 
           <Heading
             as="h3"
-            className="u-margin-bottom-sm"
+            className="mb-2"
             size="title-md"
             variant="neutral-strong"
           >
             Essential Questions
           </Heading>
-          <Text as="div" className="u-margin-bottom-xl">
+          <Text as="div" className="mb-8">
```

Comment:
Based on the [chromatic page for the project overview page](https://www.chromatic.com/test?appId=61313967cde49b003ae2a860&id=6334b5ea26cfcb769c19bfdb), I think this one needs `!important` too.
```suggestion
          <Text as="div" className="!mb-8">
```